src/Ovp/ArtifactUrlProvider.php

```diff
@@ -73,6 +75,42 @@ public function getTranscription(AssetId $id): string
         return $this->getPartial($id) . '-transcribed.json';
     }
 
+    public function getUrl($id, string $type): ?string
```

Comment:
@digitalartcrew nit: i gather that you put this here because it is in alphabetical order, but imo its weird to have it right in the middle of the others that just do getPartial. note that getPartial itself is not in alphabetical order because its a different "kind" of method in a way